pandas-stubs/_typing.pyi

```diff
@@ -135,8 +135,12 @@ Scalar = Union[
     str,
     bytes,
     datetime.date,
+    datetime.datetime,
     datetime.timedelta,
+    bool,
     complex,
```

Comment:
Can we bring back `int` and `float` here as well?

Reply:
Added

Reply:
added ignore for

> pandas-stubs/core/groupby/generic.pyi:145: error: Overloaded function signatures 1 and 3 overlap with incompatible return types  [misc]

that was hidden by https://github.com/python/mypy/issues/13775